apps/jsonrpc2/lib/bridge/sync.ex

```diff
@@ -72,4 +72,34 @@ defmodule JSONRPC2.Bridge.Sync do
       _ -> nil
     end
   end
+
+  def get_block_transaction_count_by_number(number) do
+    state_trie = get_last_sync_state().trie
+
+    case Block.get_block_by_number(number, state_trie) do
+      {:ok, block} ->
+        block.transactions
+        |> Enum.count()
+        |> :binary.encode_unsigned()
+        |> Exth.encode_hex()
```

Comment:
Do we want an `Exth.encode_unsigned_hex` since I'm a little unsure what the returned value should look like. Is this standard Hex or that weird-variety used by Web3 libraries (e.g. with *all* leading zeros removed)?